### PR TITLE
feat(AU-2115): use ATLAS_OPTIONS for pull_translations command

### PR DESCRIPTION
**Pull request by Rodra**

### CODE REVIEW

**Review by brian-smith-tcril — CHANGES_REQUESTED** [review]
Great catch! Thank you so much for this PR!

It seems the CI failure is [unrelated to this PR](https://github.com/openedx/frontend-app-ora/actions/runs/9738041863/job/26871007172) so we can disregard that.

I would like to request the changes to the `.json` files in `src/i18n/messages` be reverted, as those files are pulled from the `openedx-translations` repo via `atlas` at build time instead.

**Review by brian-smith-tcril — APPROVED** [review]

**Comment by Rodra** [comment]
@brian-smith-tcril  thank you for jumping in and check my PR. I'm not familiar with the fork process here, how can I get this merged ?

**Comment by Rodra** [comment]
@brian-smith-tcril still waiting for this to be merged since I don't have write access.

**Pull request merged**
